### PR TITLE
French basic applications

### PR DESCRIPTION
**Pull request by hchiam**
Just getting some work in before my branch gets too stale.

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
...idéale pour aider à résoudre ce problème.\n\nTâches à accomplir:\n- Apporter des modifications à la disposition, au design ou à la fonctionnalité de l'interface utilisateur\n- Demander des ressources ou un soutien supplémentaire si nécessaire\n- Fournir un calendrier estimé pour la fin des modifications\n\nRédigez une réponse par courrier électronique de Mario en utilisant le résumé ci-dessus :" initial-response="Cher John,\n\nJe vous remercie de me contacter au sujet des problèmes d'interface utilisateur avec le logiciel sur lequel je travaille. Je comprends l'importance de veiller à ce que l'interface utilisateur soit intuitive et facile à utiliser, et je suis heureux de vous aider.\n\nJe commencerai à apporter des modifications à la mise en page, au design et à la fonctionnalité de l'interface utilisateur pour résoudre les problèmes que vous avez mentionnés. Si j'ai besoin de ressources ou de soutien supplémentaires, je vous ferai savoir. Je fournirai également une estimation du délai de réalisation des modifications.\n\nN'hésitez pas à me contacter si vous avez des questions ou des préoccupations.\n\nCordialement,\nMario" max-tokens="256" box-rows="18" model-temp="0" top-p="0">
```
Comment (trigaten) [inline i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:47]: Can you use the new embeds site: http://embed.learnprompting.org

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## E-mail de base pour un jour de maladie
+
+Imaginez que vous vous réveillez malade un jour et que vous ne pouvez pas aller travailler (ou que vous ne voulez tout simplement pas 😈). Voici un prompt simple qui écrit un e-mail à votre patron pour lui dire que vous êtes malade.
+
+<div trydyno-embed="" openai-model="text-davinci-003" initial-prompt="Mon Nom : Mario\nNom du patron : John\n\nÉcrire un courriel à mon patron indiquant que je serai absent aujourd'hui car je suis malade. " initial-response="Objet : Absent - Malade\n\nCher John,\n\nJe vous écris pour vous informer que je serai absent aujourd'hui en raison de ma maladie. Je m'excuse pour tout inconvénient que cela pourrait causer.\n\nJe vais vous tenir au courant de mes progrès et je serai de retour au bureau dès que je me sentirai mieux.\n\nMerci pour votre compréhension.\n\nCordialement,\n\nMario" max-tokens="256" box-rows="8" model-temp="0" top-p="0">
+    <noscript>Échec du chargement de Dyno Embed : JavaScript doit être activé</noscript>
+</div>
```
Comment (hchiam) [inline i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:15]:
```suggestion
<iframe
    src="https://embed.learnprompting.org/embed?config=eyJ0b3BQIjowLCJ0ZW1wZXJhdHVyZSI6MCwibWF4VG9rZW5zIjoyNTYsIm91dHB1dCI6Ik9iamV0IDogQWJzZW50IC0gTWFsYWRlXFxuXFxuQ2hlciBKb2huLFxcblxcbkplIHZvdXMg6WNyaXMgcG91ciB2b3VzIGluZm9ybWVyIHF1ZSBqZSBzZXJhaSBhYnNlbnQgYXVqb3VyZCdodWkgZW4gcmFpc29uIGRlIG1hIG1hbGFkaWUuIEplIG0nZXhjdXNlIHBvdXIgdG91dCBpbmNvbnbpbmllbnQgcXVlIGNlbGEgcG91cnJhaXQgY2F1c2VyLlxcblxcbkplIHZhaXMgdm91cyB0ZW5pciBhdSBjb3VyYW50IGRlIG1lcyBwcm9ncuhzIGV0IGplIHNlcmFpIGRlIHJldG91ciBhdSBidXJlYXUgZOhzIHF1ZSBqZSBtZSBzZW50aXJhaSBtaWV1eC5cXG5cXG5NZXJjaSBwb3VyIHZvdHJlIGNvbXBy6WhlbnNpb24uXFxuXFxuQ29yZGlhbGVtZW50LFxcblxcbk1hcmlvIiwicHJvbXB0IjoiTW9uIE5vbSA6IE1hcmlvXFxuTm9tIGR1IHBhdHJvbiA6IEpvaG5cXG5cXG7JY3JpcmUgdW4gY291cnJpZWwg4CBtb24gcGF0cm9uIGluZGlxdWFudCBxdWUgamUgc2VyYWkgYWJzZW50IGF1am91cmQnaHVpIGNhciBqZSBzdWlzIG1hbGFkZS4gIiwibW9kZWwiOiJ0ZXh0LWRhdmluY2ktMDAzIn0%3D"
    style={{width:"100%", height:"500px", border:"0", borderRadius:"4px", overflow:"hidden"}}
    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
></iframe>
```

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Cet e-mail fonctionne, mais il est assez ennuyeux. Égayons-le un peu !
+
+## Modificateurs/instructions de style
+
+Il est très facile de modifier le style de l'e-mail. Par exemple, vous pouvez demander à l'IA d'être "humoristique" ou lui donner l'instruction d'inclure "une raison drôle".
+
+<div trydyno-embed="" openai-model="text-davinci-003" initial-prompt="Mon nom: Mario\nNom du patron : John\n\nÉcrivez un e-mail humoristique mais professionnel à mon patron pour lui dire que je serai absent du bureau aujourd'hui car je suis malade. \nSoyez concis et drôle. Incluez une raison amusante:" initial-response="Cher John,\n\nJe vous écris pour vous informer que je serai absent du bureau aujourd'hui. Je sais que je suis censé travailler, mais je n'ai pas pu résister à l'appel de la sirène de la boulangerie locale. Je suis sûr que vous comprenez.\n\nJe serai de retour au bureau demain et je rattraperai le temps perdu.\n\nMerci,\nMario" max-tokens="256" box-rows="10" model-temp="0.7" top-p="1">
+<noscript>Échec du chargement de Dyno Embed : JavaScript doit être activé</noscript>
+</div>
+<br/>
```
Comment (hchiam) [inline i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:26]:
```suggestion
<iframe
    src="https://embed.learnprompting.org/embed?config=eyJ0b3BQIjoxLCJ0ZW1wZXJhdHVyZSI6MC43LCJtYXhUb2tlbnMiOjI1Niwib3V0cHV0IjoiaGVyIEpvaG4sXFxuXFxuSmUgdm91cyDpY3JpcyBwb3VyIHZvdXMgaW5mb3JtZXIgcXVlIGplIHNlcmFpIGFic2VudCBkdSBidXJlYXUgYXVqb3VyZCdodWkuIEplIHNhaXMgcXVlIGplIHN1aXMgY2Vuc%2BkgdHJhdmFpbGxlciwgbWFpcyBqZSBuJ2FpIHBhcyBwdSBy6XNpc3RlciDgIGwnYXBwZWwgZGUgbGEgc2ly6G5lIGRlIGxhIGJvdWxhbmdlcmllIGxvY2FsZS4gSmUgc3VpcyBz%2B3IgcXVlIHZvdXMgY29tcHJlbmV6LlxcblxcbkplIHNlcmFpIGRlIHJldG91ciBhdSBidXJlYXUgZGVtYWluIGV0IGplIHJhdHRyYXBlcmFpIGxlIHRlbXBzIHBlcmR1Llxcblxcbk1lcmNpLFxcbk1hcmlvIiwicHJvbXB0IjoiTW9uIG5vbTogTWFyaW9cXG5Ob20gZHUgcGF0cm9uIDogSm9oblxcblxcbsljcml2ZXogdW4gZS1tYWlsIGh1bW9yaXN0aXF1ZSBtYWlzIHByb2Zlc3Npb25uZWwg4CBtb24gcGF0cm9uIHBvdXIgbHVpIGRpcmUgcXVlIGplIHNlcmFpIGFic2VudCBkdSBidXJlYXUgYXVqb3VyZCdodWkgY2FyIGplIHN1aXMgbWFsYWRlLiBcXG5Tb3lleiBjb25jaXMgZXQgZHL0bGUuIEluY2x1ZXogdW5lIHJhaXNvbiBhbXVzYW50ZToiLCJtb2RlbCI6InRleHQtZGF2aW5jaS0wMDMifQ%3D%3D"
    style={{width:"100%", height:"500px", border:"0", borderRadius:"4px", overflow:"hidden"}}
    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
></iframe>
```

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+Voici un autre exemple qui est plus sérieux/professionnel.
+
+<div trydyno-embed="" openai-model="text-davinci-003" initial-prompt="Mon nom : Mario\nNom de mon patron : John\n\nRédigez un e-mail **sérieux** et professionnel à mon patron indiquant que je serai absent du bureau aujourd'hui car je suis malade.\nSois concis :" initial-response="Objet : Absence - Malade\n\nCher John,\n\nJe vous informe que je serai absent du bureau aujourd'hui en raison de ma maladie. Je vous prie de m'excuser pour toute gêne occasionnée.\n\nMerci de votre compréhension.\n\nCordialement,\nMario" max-tokens="256" box-rows="8" model-temp="0" top-p="0">
+<noscript>Échec du chargement de Dyno Embed : JavaScript doit être activé</noscript>
+</div>
+<br/>
```
Comment (hchiam) [inline i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:33]:
```suggestion
<iframe
    src="https://embed.learnprompting.org/embed?config=eyJ0b3BQIjowLCJ0ZW1wZXJhdHVyZSI6MCwibWF4VG9rZW5zIjoyNTYsIm91dHB1dCI6Ik9iamV0IDogQWJzZW5jZSAtIE1hbGFkZVxcblxcbkNoZXIgSm9obixcXG5cXG5KZSB2b3VzIGluZm9ybWUgcXVlIGplIHNlcmFpIGFic2VudCBkdSBidXJlYXUgYXVqb3VyZCdodWkgZW4gcmFpc29uIGRlIG1hIG1hbGFkaWUuIEplIHZvdXMgcHJpZSBkZSBtJ2V4Y3VzZXIgcG91ciB0b3V0ZSBn6m5lIG9jY2FzaW9ubullLlxcblxcbk1lcmNpIGRlIHZvdHJlIGNvbXBy6WhlbnNpb24uXFxuXFxuQ29yZGlhbGVtZW50LFxcbk1hcmlvIiwicHJvbXB0IjoiTW9uIG5vbSA6IE1hcmlvXFxuTm9tIGRlIG1vbiBwYXRyb24gOiBKb2huXFxuXFxuUulkaWdleiB1biBlLW1haWwgKipz6XJpZXV4KiogZXQgcHJvZmVzc2lvbm5lbCDgIG1vbiBwYXRyb24gaW5kaXF1YW50IHF1ZSBqZSBzZXJhaSBhYnNlbnQgZHUgYnVyZWF1IGF1am91cmQnaHVpIGNhciBqZSBzdWlzIG1hbGFkZS5cXG5Tb2lzIGNvbmNpcyA6IiwibW9kZWwiOiJ0ZXh0LWRhdmluY2ktMDAzIn0%3D"
    style={{width:"100%", height:"500px", border:"0", borderRadius:"4px", overflow:"hidden"}}
    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
></iframe>
```

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md (hunk 1 above)
Comment (hchiam) [reply i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:47]: will do

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+<noscript>Échec du chargement de Dyno Embed : JavaScript doit être activé</noscript>
+</div>
```
Comment (hchiam) [inline i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:43]:
```suggestion
<iframe
    src="https://embed.learnprompting.org/embed?config=eyJ0b3BQIjowLCJ0ZW1wZXJhdHVyZSI6MCwibWF4VG9rZW5zIjoyNTYsIm91dHB1dCI6IlLpc3Vt6SA6IEpvaG4gZGVtYW5kZSBsJ2FpZGUgZGUgTWFyaW8gcG91ciBy6XNvdWRyZSBsZXMgcHJvYmzobWVzIGQnaW50ZXJmYWNlIHV0aWxpc2F0ZXVyIGR1IGxvZ2ljaWVsIHN1ciBsZXF1ZWwgTWFyaW8gdHJhdmFpbGxlLiBMZXMgcHJvYmzobWVzIGNvbmNlcm5lbnQgbGEgZGlmZmljdWx06SBkJ2FjY%2BlkZXIg4CBjZXJ0YWluZXMgZm9uY3Rpb25uYWxpdOlzIGV0IGwnaW50ZXJmYWNlIHV0aWxpc2F0ZXVyIHF1aSBuJ2VzdCBwYXMgYXNzZXogaW50dWl0aXZlLiBMJ2V4cGVydGlzZSBkZSBNYXJpbyBlbiBk6XZlbG9wcGVtZW50IGRlIGxvZ2ljaWVscyBldCBzYSBmYW1pbGlhcml06SBhdmVjIGxlIGxvZ2ljaWVsIGZvbnQgZGUgbHVpIGxhIHBlcnNvbm5lIGlk6WFsZSBwb3VyIGFpZGVyIOAgculzb3VkcmUgY2UgcHJvYmzobWUuXFxuXFxuVOJjaGVzIOAgYWNjb21wbGlyOlxcbi0gQXBwb3J0ZXIgZGVzIG1vZGlmaWNhdGlvbnMg4CBsYSBkaXNwb3NpdGlvbiwgYXUgZGVzaWduIG91IOAgbGEgZm9uY3Rpb25uYWxpdOkgZGUgbCdpbnRlcmZhY2UgdXRpbGlzYXRldXJcXG4tIERlbWFuZGVyIGRlcyByZXNzb3VyY2VzIG91IHVuIHNvdXRpZW4gc3VwcGzpbWVudGFpcmUgc2kgbuljZXNzYWlyZVxcbi0gRm91cm5pciB1biBjYWxlbmRyaWVyIGVzdGlt6SBwb3VyIGxhIGZpbiBkZXMgbW9kaWZpY2F0aW9ucyIsInByb21wdCI6IkNoZXIgTWFyaW8sXFxuXFxuSidlc3DocmUgcXVlIHR1IHZhcyBiaWVuLiBKZSB0J%2BljcmlzIHBvdXIgdGUgZG9ubmVyIGRlcyBtaXNlcyDgIGpvdXIgc3VyIG5vdHJlIGVudHJlcHJpc2UgZXQgdGUgZGVtYW5kZXIgdG9uIGFpZGUgcG91ciBy6XNvdWRyZSB1biBwcm9ibOhtZSBhdmVjIGwnaW50ZXJmYWNlIHV0aWxpc2F0ZXVyIChVSSkgZHUgbG9naWNpZWwgc3VyIGxlcXVlbCB0dSB0cmF2YWlsbGVzLlxcblxcbkNvbW1lIHR1IGxlIHNhaXMsIG5vdHJlIGVudHJlcHJpc2Ugc2UgY29uY2VudHJlIHN1ciBsYSBmb3Vybml0dXJlIGQndW5lIGV4cOlyaWVuY2UgY29udml2aWFsZSDgIG5vcyBjbGllbnRzIGV0IG5vdXMgYXZvbnMgaWRlbnRpZmnpIGRlcyBwcm9ibOhtZXMgYXZlYyBsJ2ludGVyZmFjZSB1dGlsaXNhdGV1ciBkdSBsb2dpY2llbC4gRW4gcGFydGljdWxpZXIsIG5vdXMgYXZvbnMgcmXndSBkZXMgY29tbWVudGFpcmVzIGRlIG5vcyB1dGlsaXNhdGV1cnMgaW5kaXF1YW50IHF1J2lscyBvbnQgZHUgbWFsIOAgYWNj6WRlciDgIGNlcnRhaW5lcyBmb25jdGlvbm5hbGl06XMgZXQgcXVlIGwnaW50ZXJmYWNlIHV0aWxpc2F0ZXVyIG4nZXN0IHBhcyBhc3NleiBpbnR1aXRpdmUuXFxuXFxuTm91cyBhdm9ucyBiZXNvaW4gZGUgdG9uIGFpZGUgcG91ciBy6XNvdWRyZSBjZXMgcHJvYmzobWVzIGV0IGFt6Wxpb3JlciBsJ2ludGVyZmFjZSB1dGlsaXNhdGV1ciBkdSBsb2dpY2llbC4gQ2VsYSBwZXV0IGltcGxpcXVlciBkZXMgbW9kaWZpY2F0aW9ucyDgIGxhIGRpc3Bvc2l0aW9uLCBhdSBkZXNpZ24gb3Ug4CBsYSBmb25jdGlvbm5hbGl06SBkZSBsJ2ludGVyZmFjZSB1dGlsaXNhdGV1ci4gTm91cyBwZW5zb25zIHF1ZSB0b24gZXhwZXJ0aXNlIGVuIGTpdmVsb3BwZW1lbnQgZGUgbG9naWNpZWxzIGV0IHRhIGZhbWlsaWFyaXTpIGF2ZWMgbGUgbG9naWNpZWwgZmVyb250IGRlIHRvaSBsYSBwZXJzb25uZSBpZOlhbGUgcG91ciBub3VzIGFpZGVyIGRhbnMgY2V0dGUgdOJjaGUuXFxuXFxuU2kgdHUgYXMgYmVzb2luIGRlIHJlc3NvdXJjZXMgb3UgZGUgc291dGllbiBzdXBwbOltZW50YWlyZXMgcG91ciBtZW5lciDgIGJpZW4gY2V0dGUgdOJjaGUsIG4naOlzaXRlIHBhcyDgIG1lIGxlIGZhaXJlIHNhdm9pci4gRGUgcGx1cywgaidhcHBy6WNpZXJhaXMgc2kgdHUgcG91dmFpcyBtZSBmb3VybmlyIHVuIGNhbGVuZHJpZXIgZXN0aW3pIHBvdXIgcXVhbmQgdHUgcHLpdm9pcyBxdWUgY2VzIG1vZGlmaWNhdGlvbnMgc2Vyb250IHRlcm1pbullcy5cXG5cXG5TaSB0dSBhcyBkZXMgcXVlc3Rpb25zIG91IGJlc29pbiBkZSBwbHVzIGRlIGNsYXJpZmljYXRpb24sIG4naOlzaXRlIHBhcyDgIG1lIGNvbnRhY3Rlci5cXG5cXG5NZXJjaSBwb3VyIHRvbiB0cmF2YWlsIGFjaGFybukgZXQgdGEgZOl2b3Rpb24gZW52ZXJzIG5vdHJlIGVudHJlcHJpc2UuIEonYXR0ZW5kcyBhdmVjIGltcGF0aWVuY2UgZGUgdGVzIG5vdXZlbGxlcyBiaWVudPR0LlxcblxcbkNvcmRpYWxlbWVudCxcXG5Kb2huXFxuXFxuR%2Blu6XJlciB1biBy6XN1bekgZGUgY2VjaSBldCB1bmUgbGlzdGUgZGUgdOJjaGVzIOAgYWNjb21wbGlyLiIsIm1vZGVsIjoidGV4dC1kYXZpbmNpLTAwMyJ9"
    style={{width:"100%", height:"500px", border:"0", borderRadius:"4px", overflow:"hidden"}}
    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
></iframe>
```

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
```
Comment (hchiam) [inline i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:49]:
```suggestion
<iframe
    src="https://embed.learnprompting.org/embed?config=eyJ0b3BQIjowLCJ0ZW1wZXJhdHVyZSI6MCwibWF4VG9rZW5zIjoyNTYsIm91dHB1dCI6IkNoZXIgSm9obixcXG5cXG5KZSB2b3VzIHJlbWVyY2llIGRlIG1lIGNvbnRhY3RlciBhdSBzdWpldCBkZXMgcHJvYmzobWVzIGQnaW50ZXJmYWNlIHV0aWxpc2F0ZXVyIGF2ZWMgbGUgbG9naWNpZWwgc3VyIGxlcXVlbCBqZSB0cmF2YWlsbGUuIEplIGNvbXByZW5kcyBsJ2ltcG9ydGFuY2UgZGUgdmVpbGxlciDgIGNlIHF1ZSBsJ2ludGVyZmFjZSB1dGlsaXNhdGV1ciBzb2l0IGludHVpdGl2ZSBldCBmYWNpbGUg4CB1dGlsaXNlciwgZXQgamUgc3VpcyBoZXVyZXV4IGRlIHZvdXMgYWlkZXIuXFxuXFxuSmUgY29tbWVuY2VyYWkg4CBhcHBvcnRlciBkZXMgbW9kaWZpY2F0aW9ucyDgIGxhIG1pc2UgZW4gcGFnZSwgYXUgZGVzaWduIGV0IOAgbGEgZm9uY3Rpb25uYWxpdOkgZGUgbCdpbnRlcmZhY2UgdXRpbGlzYXRldXIgcG91ciBy6XNvdWRyZSBsZXMgcHJvYmzobWVzIHF1ZSB2b3VzIGF2ZXogbWVudGlvbm7pcy4gU2kgaidhaSBiZXNvaW4gZGUgcmVzc291cmNlcyBvdSBkZSBzb3V0aWVuIHN1cHBs6W1lbnRhaXJlcywgamUgdm91cyBmZXJhaSBzYXZvaXIuIEplIGZvdXJuaXJhaSDpZ2FsZW1lbnQgdW5lIGVzdGltYXRpb24gZHUgZOlsYWkgZGUgculhbGlzYXRpb24gZGVzIG1vZGlmaWNhdGlvbnMuXFxuXFxuTido6XNpdGV6IHBhcyDgIG1lIGNvbnRhY3RlciBzaSB2b3VzIGF2ZXogZGVzIHF1ZXN0aW9ucyBvdSBkZXMgcHLpb2NjdXBhdGlvbnMuXFxuXFxuQ29yZGlhbGVtZW50LFxcbk1hcmlvIiwicHJvbXB0IjoiUulzdW3pIDogSm9obiBkZW1hbmRlIGwnYWlkZSBkZSBNYXJpbyBwb3VyIHLpc291ZHJlIGxlcyBwcm9ibOhtZXMgZCdpbnRlcmZhY2UgdXRpbGlzYXRldXIgZHUgbG9naWNpZWwgc3VyIGxlcXVlbCBNYXJpbyB0cmF2YWlsbGUuIExlcyBwcm9ibOhtZXMgY29uY2VybmVudCBsYSBkaWZmaWN1bHTpIGQnYWNj6WRlciDgIGNlcnRhaW5lcyBmb25jdGlvbm5hbGl06XMgZXQgbCdpbnRlcmZhY2UgdXRpbGlzYXRldXIgcXVpIG4nZXN0IHBhcyBhc3NleiBpbnR1aXRpdmUuIEwnZXhwZXJ0aXNlIGRlIE1hcmlvIGVuIGTpdmVsb3BwZW1lbnQgZGUgbG9naWNpZWxzIGV0IHNhIGZhbWlsaWFyaXTpIGF2ZWMgbGUgbG9naWNpZWwgZm9udCBkZSBsdWkgbGEgcGVyc29ubmUgaWTpYWxlIHBvdXIgYWlkZXIg4CBy6XNvdWRyZSBjZSBwcm9ibOhtZS5cXG5cXG5U4mNoZXMg4CBhY2NvbXBsaXI6XFxuLSBBcHBvcnRlciBkZXMgbW9kaWZpY2F0aW9ucyDgIGxhIGRpc3Bvc2l0aW9uLCBhdSBkZXNpZ24gb3Ug4CBsYSBmb25jdGlvbm5hbGl06SBkZSBsJ2ludGVyZmFjZSB1dGlsaXNhdGV1clxcbi0gRGVtYW5kZXIgZGVzIHJlc3NvdXJjZXMgb3UgdW4gc291dGllbiBzdXBwbOltZW50YWlyZSBzaSBu6WNlc3NhaXJlXFxuLSBGb3VybmlyIHVuIGNhbGVuZHJpZXIgZXN0aW3pIHBvdXIgbGEgZmluIGRlcyBtb2RpZmljYXRpb25zXFxuXFxuUulkaWdleiB1bmUgculwb25zZSBwYXIgY291cnJpZXIg6WxlY3Ryb25pcXVlIGRlIE1hcmlvIGVuIHV0aWxpc2FudCBsZSBy6XN1bekgY2ktZGVzc3VzIDoiLCJtb2RlbCI6InRleHQtZGF2aW5jaS0wMDMifQ%3D%3D"
    style={{width:"100%", height:"500px", border:"0", borderRadius:"4px", overflow:"hidden"}}
    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
></iframe>
```

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+<div trydyno-embed="" openai-model="text-davinci-003" initial-prompt="Company name: Nightfall\nMy name: Max\nWrite a cold outreach email to this founder, named Pranav, pitching him our product, which is a neo-CRM. Make the email formal, yet approachable." initial-response="Subject: A modern CRM to help your business grow\n\nDear Pranav,\n\nMy name is Max and I'm the founder of Nightfall, a neo-CRM that helps businesses streamline their customer relationship management.\n\nWe understand that managing customer relationships can be a challenge, especially as your business grows. That's why we created Nightfall, a modern CRM that helps you stay organized and connected with your customers.\n\nNightfall offers a range of features, including automated customer segmentation, personalized customer journeys, and real-time analytics. We also provide 24/7 customer support to ensure that you get the most out of our product.\n\nI'd love to discuss how Nightfall can help your business grow. Would you be available for a call sometime this week?\n\nI look forward to hearing from you.\n\nBest,\nMax" max-tokens="256" box-rows="10" model-temp="0.0" top-p="0">
```
Comment (hchiam) [inline i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:65]: will continue translating later to avoid a stale branch

**Review by hchiam — COMMENTED** [review]
Ready for review.

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
...ounder Resources at Startup Shell\nExploring the beauty of mathematics with the practicality of computer science\n\nWrite a cold outreach email to this founder, pitching him our product, Nightfall, which is a neo-CRM. My name is Max. Make the email formal, yet approachable. Mention relevant details from his LinkedIn information above to help pitch the product." initial-response="Dear Pranav,\n\nMy name is Max and I am reaching out to you because I believe our product, Nightfall, could be a great fit for you. As a co-founder of Strive and Director of Founder Resources at Startup Shell, I'm sure you understand the importance of having a reliable CRM to help manage customer relationships.\n\nNightfall is a neo-CRM that helps founders and entrepreneurs like yourself to easily manage customer relationships and track customer data. Our product is designed to be user-friendly and intuitive, so you can focus on growing your business without worrying about the technical details.\n\nI think you would find Nightfall to be a great asset to your business. If you're interested, I'd love to schedule a call to discuss further.\n\nBest,\nMax" max-tokens="256" box-rows="39" model-temp="0" top-p="0">
```
Comment (trigaten) [inline i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:75]: please replace w/ new embed

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md (hunk 7 above)
Comment (trigaten) [reply i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:65]: sounds good, ty

File: i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md (hunk 8 above)
Comment (hchiam) [reply i18n/fr/docusaurus-plugin-content-docs/current/basic_applications/writing_emails.md:75]: done

**Comment by hchiam** [comment]
ready for review

**Pull request merged**